fp.pretty/Interface_UART_RS485_Renesas_SMD.pretty/TDFN-8_B2.0x3.0x0.75mm_P0.5.kicad_mod

```diff
@@ -30,8 +30,8 @@
   (pad 3 smd rect (at 0.25 1.5) (size 0.25 0.4) (layers F.Cu F.Paste F.Mask))
   (pad 5 smd rect (at 0.75 -1.5) (size 0.25 0.4) (layers F.Cu F.Paste F.Mask))
   (pad 4 smd rect (at 0.75 1.5) (size 0.25 0.4) (layers F.Cu F.Paste F.Mask))
-  (model "${KIPRJMOD}/3D Models/User Library-TDFN-8-1.STEP"
-    (offset (xyz -0.6 -1.5 0))
+  (model ${KISYS3DMOD}/kicad-packages3D/blob/master/Package_DFN_QFN.3dshapes/DFN-8-1EP_2x3mm_P0.5mm_EP0.61x2.2mm.step
```

Comment:
Just need to adjust this path to:
* remove the `kicad-packages3D/blob/master/` part
* point to the `.wrl` instead of `.step` (see point 8 here https://kicad-pcb.org/libraries/klc/F9.3/)

Reply:
Can you also move this file to:

fp.pretty/TDFN-8_B2.0x3.0x0.75mm_P0.5.kicad_mod

Reply:
Done.